app/src/renderer/css/editor.css

```diff
@@ -19,7 +19,6 @@ html { height: 100%; }
 body {
   background-color: rgba(0, 0, 0, .6);
   -webkit-app-region: drag;
-  /*cursor: -webkit-grab;*/
```

Comment:
I think this is an error in the `master` branch?

Reply:
Can you elaborate? If it is an error, you should just remove it.

Reply:
Yes, that's what I did 👍

Reply:
Never mind me. I was confused and hungry.